modules/installation-gcp-user-infra-completing.adoc

```diff
@@ -0,0 +1,82 @@
+// Module included in the following assemblies:
```

Comment:
At some point, we should improve the reuse with https://github.com/openshift/openshift-docs/pull/17190/files#diff-b4f4624fd26f9cf529c5e626f5ba805fR7